tests/travis/run_test.sh

```diff
@@ -79,23 +79,40 @@ fi
 
 if [ ${TASK} == "r_test" ]; then
     set -e
-    export _R_CHECK_TIMINGS_=0
-    export R_BUILD_ARGS="--no-build-vignettes --no-manual"
-    export R_CHECK_ARGS="--no-vignettes --no-manual"
-    if [ ${TRAVIS_OS_NAME} == "osx" ]; then
-        # Work-around to fix "gfortran command not found" error
-        sudo ln -s $(which gfortran-7) /usr/local/bin/gfortran
-        sudo mkdir -p /usr/local/gfortran/lib/gcc/x86_64-apple-darwin15
-        sudo ln -s /usr/local/lib/gcc/7 /usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0
-    fi
-
-    curl -OL https://raw.githubusercontent.com/craigcitro/r-travis/master/scripts/travis-tool.sh
-    chmod 755 ./travis-tool.sh
-    ./travis-tool.sh bootstrap
+
     make Rpack
     cd ./xgboost
-    ../travis-tool.sh install_deps
-    ../travis-tool.sh run_tests
+
+    # Install package deps
+    Rscript -e "install.packages('devtools', repos = 'http://cloud.r-project.org')"
+    Rscript -e \
+        "devtools::install_deps( \
+            repos = 'http://cloud.r-project.org',\
+            upgrade = 'never', \
+            dependencies = c('Depends', 'Imports') \
+        )"
+
+    # install suggested packages separately to avoid huge build times
+    Rscript -e "install.packages(c('DiagrammeR', 'Ckmeans.1d.dp', 'vcd'), repos = 'https://cloud.r-project.org')"
```

Comment:
@hcho3 for my own knowledge, why was it necessary to change to `cloud.r-project.org` from the RStudio mirror?

Reply:
From [this page](https://cran.r-project.org/mirrors.html): "Automatic redirection to servers worldwide, currently sponsored by Rstudio"

Reply:
thank you